include/render/Mesh.h

```diff
@@ -0,0 +1,28 @@
+
+
+#ifndef MESH_H
+#define MESH_H
+#include "Model.h"
+
+class Mesh
+{
+  
+public:
+
+  Mesh(Model*, unsigned int volume_id);
+  virtual ~Mesh();
+
+  unsigned int get_volume_id(); 
+
+  void set_volume_id(unsigned int val); 
+
+  Model& get_model(); 
```

Comment:
Same here, shouldn't this be a constant reference? Do you want the model to be modified from outside world?

Reply:
I agree, I am solo dev on this project and wasnt expecting this kind of feedback.   https://github.com/brown-ccv/VR-Volumeviewer/issues/58
Let's move this change to another PR in order to not increasing the number of modified on this one. At this moment the soft is stable enough.